.github/workflows/publish.yml

```diff
@@ -158,6 +158,12 @@ jobs:
         npm run build
       working-directory: packages/react-native-icons 
 
+    - uses: JS-DevTools/npm-publish@v1
+      with:
+        token: ${{ secrets.NPM_TOKEN }}
+        access: public
+        package: packages/svg-icons/package.json
+
     - uses: JS-DevTools/npm-publish@v1
       with:
         token: ${{ secrets.NPM_TOKEN }}
```

Comment:
please remove lines 173-178, as the react-native-icons publish on line 200 below would suffice

Reply:
Done.